### PR TITLE
print_summary option to benchmark_client to parse benchmark results.

### PR DESCRIPTION
**Pull request by psakamoori**
Added print_summary command arg to benchmark_client app to summarize benchmark results as shown below 

```
docker run --network host benchmark_client -a localhost -r 8000 -m face-detection-retail-0005 -p 9000 -s 2 3 300 300 -t 20 -u 2 -w 10  -ps
Client 2.7
NO_PROXY=localhost no_proxy=localhost python3 /ovms_benchmark_client/main.py -a localhost -r 8000 -m face-detection-retail-0005 -p 9000 -s 2 3 300 300 -t 20 -u 2 -w 10 -ps
          XI worker: start workload...

### Benchmark Parameters ###
 Model: face-detection-retail-0005
 Input shape: ['2', '3', '300', '300']
 Request concurrency: 1
 Test Duration (s): Total (t) : 20.00 | Warmup (u): 2.00 | Window (w): 10.00

### Benchmark Summary ###
 ## General Metrics ##
 Duration(s): Total: 20.01 | Window: 10.01
 Batches: Total: 1781 | Window: 891

 ## Latency Metrics (ms) ##
 Mean: 11.20 | stdev: 0.74 | p50: 12.78 | p90: 15.26 | p95: 15.56

 ## Throughput Metrics (fps) ##
 Frame Rate (FPS): Brutto : 89.01 | Netto: 89.24 
 Batch Rate (batches/s): Brutto: 89.01 | Netto: 89.24
```

Currently print_all results need manually parsing to understand the over-all performance summary. This PR helps to automate the print_all results parsing and summarize the performance benchmark results.

### CODE REVIEW

File: demos/benchmark/python/main.py [hunk 1]
```diff
@@ -333,7 +335,79 @@ def __getattr__(self, attr):
...
+
+                sys.stdout.write(f" | {p}: {common_results[qv]*1000:.2f}")
+            sys.stdout.write("\n")
+            sys.stdout.write("\n ## Throughput Metrics (fps) ##\n")
+
+            # Brutto: Total number of frames processed or produced per second by a system or application.
+            # It doesn't take into account any overhead or inefficiencies in the system
+            # Netto: Effective or net frame rate, which is the frame rate adjusted for any overhead,
+            # delays, or processing inefficiencies in the system
+            sys.stdout.write(f" Frame Rate (FPS): Brutto : {common_results['brutto_frame_rate']:.2f}")
```
Comment (ravi9) [inline demos/benchmark/python/main.py:408]:
```suggestion
            sys.stdout.write(f" Frame Rate (FPS): Brutto: {common_results['brutto_frame_rate']:.2f}")
```

File: demos/benchmark/python/main.py [hunk 2]
```diff
@@ -333,7 +335,79 @@ def __getattr__(self, attr):
...
+            sys.stdout.write(f" Input shape: {inp_shape}\n")
+        if 'submetrics' in common_results:
+            total_clients = common_results["submetrics"]
+        else:
+            total_clients = 1
+
+        sys.stdout.write(f" Request concurrency: {total_clients}\n")
+        if xargs['duration']:
+            total_t = float(xargs['duration'])
+            sys.stdout.write(f" Test Duration (s): Total (t) : {total_t:.2f}")
```
Comment (ravi9) [inline demos/benchmark/python/main.py:359]:
```suggestion
            sys.stdout.write(f" Test Duration (s): Total (t): {total_t:.2f}")
```

File: demos/benchmark/python/main.py [hunk 3]
```diff
@@ -262,6 +262,8 @@ def __getattr__(self, attr):
                         help="database metadata configuration. default: None")
     parser.add_argument("--print_all", required=False, action="store_true",
                         help="flag to print all output")
+    parser.add_argument("-ps", "--print_summary", required=False, action="store_true",
+                        help="flag to print results summary")
```
Comment (ravi9) [inline demos/benchmark/python/main.py:266]:
```suggestion
                        help="flag to print output summary")
```

File: demos/benchmark/python/README.md [hunk 4]
```diff
@@ -326,6 +326,38 @@ NO_PROXY=localhost no_proxy=localhost python3 /ovms_benchmark_client/main.py -a
...
+### Benchmark Summary ###
+ ## General Metrics ##
+ Duration(s): Total: 20.01 | Window: 10.01
+ Batches: Total: 1781 | Window: 891
+
+ ## Latency Metrics (ms) ##
+ Mean: 11.20 | stdev: 0.74 | p50: 12.78 | p90: 15.26 | p95: 15.56
+
+ ## Throughput Metrics (fps) ##
+ Frame Rate (FPS): Brutto : 89.01 | Netto: 89.24
```
Comment (ravi9) [inline demos/benchmark/python/README.md:358]:
```suggestion
 Frame Rate (FPS): Brutto: 89.01 | Netto: 89.24
```

**Review by ravi9 — CHANGES_REQUESTED** [review]
Minor changes

**Review by dtrawins — APPROVED** [review]

File: demos/benchmark/python/README.md [hunk 5]
```diff
@@ -326,6 +326,38 @@ NO_PROXY=localhost no_proxy=localhost python3 /ovms_benchmark_client/main.py -a
...
 ```
 
+## Summarize benchmarking results
+
+Summary of the benchmark results can be viewed with command option ```-ps```
+```
+docker run --network host benchmark_client -a localhost -r 8000 -m face-detection-retail-0005 -p 9000 -s 2 3 300 300 -t 20 -u 2 -w 10 -ps
+```
+
+Sample ouptut log with results summary
```
Comment (kpapis) [inline demos/benchmark/python/README.md:336]:
```suggestion
Sample output log with results summary:
```

File: demos/benchmark/python/README.md [hunk 6]
```diff
@@ -326,6 +326,38 @@ NO_PROXY=localhost no_proxy=localhost python3 /ovms_benchmark_client/main.py -a
 ...
 ```
 
+## Summarize benchmarking results
+
+Summary of the benchmark results can be viewed with command option ```-ps```
```
Comment (kpapis) [inline demos/benchmark/python/README.md:331]:
```suggestion
Summary of the benchmark results can be viewed with command option ```-ps```.
```

**Review by kpapis — APPROVED** [review]

**Pull request merged**